src/_locales/hi/messages.json

```diff
@@ -75,84 +75,84 @@
+		"message": "आप 1 सेकंड में आपने आप पुनर्निर्देशित हो जाएंगे।"
 	},
 	"beforeNavigateOptions": {
 		"message": "क्या आप तुरंत या x सेकंड में पुनर्निर्देशित होना चाहते हैं?"
 	},
 	"beforeNavigateInstant": {
-		"message": "आपको% पर नेविगेट किया जा रहा है"
+		"message": "आपको % पर ले जाया जा रहा है।"
 	},
 	"infoLinkvertise": {
-		"message": "हमें इस वेबसाइट को बायपास करने की अनुमति नहीं है लेकिन हमने उनके सबसे कष्टप्रद कदमों को हटाने के लिए बातचीत की है।"
+		"message": "हमें इस वेबसाइट पर बाईपास करने की अनुमति नहीं है लेकिन हमने उनके सबसे कष्टप्रद चरणों को हटाने के लिए बातचीत की है।"
 	},
 	"infoFileHoster": {
-		"message": "दुर्भाग्य से, डाउनलोड सर्वर यह सुनिश्चित करेगा कि आपने फ़ाइल डाउनलोड करने की अनुमति देने से पहले इंतजार किया है।"
+		"message": "दुर्भाग्य से, डाउनलोड सर्वर यह सुनिश्चित करेगा कि आपको फ़ाइल डाउनलोड करने की अनुमति देने से पहले आपने इंतजार किया है।"
 	},
 	"infoOutdated": {
-		"message": "यह वेबसाइट यूनिवर्सल बाईपास के एक नए संस्करण द्वारा बाईपास की गई है।"
+		"message": "यह वेबसाइट FastForward के एक नए संस्करण द्वारा बाईपास की गई है।"
 	},
 	"crowdWait": {
-		"message": "दुर्भाग्य से, आप इस लिंक का सामना करने वाले पहले व्यक्ति हैं, लेकिन आपके इंतजार करने के बाद, अन्य यूनिवर्सल बायपास उपयोगकर्ताओं को अब नहीं रहना पड़ेगा।"
+		"message": "दुर्भाग्य से, आप इस लिंक का सामना करने वाले पहले व्यक्ति हैं, लेकिन आपके इंतजार करने के बाद, अन्य FastForward उपयोगकर्ताओं को इंतज़ार नहीं रहना पड़ेगा।"
 	},
 	"crowdDisabled": {
-		"message": "यह वेबसाइट भीड़ द्वारा बाईपास है, जिसे आप विकल्पों में सक्षम कर सकते हैं।"
+		"message": "यह वेबसाइट क्राउड बाईपास द्वारा बाईपास की गयी है, जिसे आप विकल्पों में सक्षम कर सकते हैं।"
 	},
 	"crowdBypassed": {
-		"message": "शायद आपको इंतज़ार ना करना पड़े"
+		"message": "शायद आपको इंतज़ार ना करना पड़े!"
 	},
 	"crowdBypassedInfo": {
-		"message": "अन्य यूनिवर्सल बाईपास उपयोगकर्ता आपके लिए पहले ही इंतजार कर चुके हैं और रिपोर्ट कर चुके हैं कि यह लिंक इसपर जाता है %"
+		"message": "अन्य FastForward उपयोगकर्ता आपके लिए पहले ही इंतजार कर चुके हैं और रिपोर्ट कर चुके हैं कि यह लिंक इस पर जाता है %"
 	},
 	"crowdBypassedTimer": {
-		"message": "इसे% सेकंड में एक नए टैब में खोला जाएगा।"
+		"message": "यह % सेकंड में एक नयी टैब में खोला जाएगा।"
 	},
 	"crowdBypassedTimerSingular": {
-		"message": "यह 1 सेकंड में एक नए टैब में खोला जाएगा।"
+		"message": "यह 1 सेकंड में एक नयी टैब में खोला जाएगा।"
 	},
 	"crowdCloseTimer": {
-		"message": "यह टैब% सेकंड में बंद हो जाएगा।"
+		"message": "यह टैब % सेकंड में बंद हो जाएगी।"
 	},
 	"crowdCloseTimerSingular": {
-		"message": "यह टैब 1 सेकंड में बंद हो जाएगा।"
+		"message": "यह टैब 1 सेकंड में बंद हो जाएगी।"
 	},
 	"cancel": {
-		"message": "रद्द करें"
+		"message": "रद्द करें।"
 	},
 	"crowdBypassedIgnore": {
 		"message": "क्या ये सही नहीं था?"
 	},
 	"crowdBypassedOptions": {
-		"message": "क्या आप चाहते हैं कि यह तुरंत एक नए टैब में या एक्स सेकंड में खोला जाए?"
+		"message": "क्या आप चाहते हैं कि यह तुरंत एक नए टैब में या x सेकंड में खोला जाए?"
```

Comment:
x ki jagah "कुछ" ?

Reply:
Nope.

Reply:
This 'x' will be replaced with something as I know.

Reply:
no it's actually just the letter 'x' on the before navigate page 
![image](https://user-images.githubusercontent.com/81463636/169851614-425b613c-fc1d-48e2-886c-d2f197026df5.png)

Reply:
it shows change in FastForward options, so if that setting is changed with any number then 'x' will be replaced by that number.
![image](https://user-images.githubusercontent.com/50015986/169941653-bb1bdb4c-0c5f-4d52-a050-c0f897e12d15.png)

I think this setting affects this 'x'

Reply:
Thanks for correcting me, I changed replaced 'x' with 'कुछ'. I hope there is no any other issue in file, if there is any then let me know.